daemon/daemon_test.go

```diff
@@ -22,8 +22,9 @@ import (
 	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/policy"
 
-	. "gopkg.in/check.v1"
 	"sync"
```

Comment:
Imports order should be `golib`, `cilium`, `3rd parties`

`sync` should be on the first group